network/endpoint_windows.go

```diff
@@ -206,6 +207,10 @@ func (nw *network) addIPv6NeighborEntryForGateway(epInfo *EndpointInfo, plc plat
 func (nw *network) configureHcnEndpoint(epInfo *EndpointInfo) (*hcn.HostComputeEndpoint, error) {
 	infraEpName, _ := ConstructEndpointID(epInfo.ContainerID, epInfo.NetNsPath, epInfo.IfName)
 
+	mac := epInfo.MacAddress.String()
```

Comment:
will these new fields be needed for every request? (primary, non-swift2.0, secondary) if not need branching here too